dnaseq_slurm_api.sh

```diff
@@ -0,0 +1,37 @@
+# sample commands to be called by stager slurm.py with variable substitution
```

Comment:
I would not inline this in Stager code but call this script. The `script` payload sent to the Slurm API would look like this
```bash
#!/usr/bin/env bash
exec /path/to/dnaseq_slurm_api.sh {filepath} {analysis_id} {family} {participant}:{linked_files}... {arguments}
```

And thus the start of this file should look something like
```bash
#!/usr/bin/env bash

set -euo pipefail

FILEPATH="$1"
ANALYSIS_ID="$2"
# remaining argument parsing
```

Reply:
got it, thanks- will modify

Reply:
changed in 9e06285b07c604af89924b40d1f44709407e1e75